# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/st/hotfix-no-termcap.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- st-0.9.2/Makefile.vanilla	2024-05-19 23:14:17.821000000 +0200
+++ st-0.9.2/Makefile	2024-05-19 23:14:27.118000000 +0200
@@ -41,7 +41,7 @@
 	mkdir -p $(DESTDIR)$(MANPREFIX)/man1
 	sed "s/VERSION/$(VERSION)/g" < st.1 > $(DESTDIR)$(MANPREFIX)/man1/st.1
 	chmod 644 $(DESTDIR)$(MANPREFIX)/man1/st.1
-	tic -sx st.info
+	#tic -sx st.info
 	@echo Please see the README file regarding the terminfo entry of st.
 
 uninstall:
